spec/controllers/ticket_dispenser/tickets/replies_controller_spec.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe TicketDispenser::Tickets::RepliesController, type: :request do
+      describe '#destroy' do
```

Comment:
This is not standard Ruby indentation. Should be two spaces.

Reply:
I'll make the necessary changes